### ISSUE TITLE
Make OSARA: Report edit/play cursor position Honour The Secondary Time Unit

### ISSUE THREAD

**Issue by MatejGolian** [jcsteh/osara#224]
This is not a must, but it would be nice to have.
I think that  the second trigger of OSARA: Report edit/play cursor position should report the current position in the secondary time unit. Currently the second trigger of this action works only when the secondary time unit is set to minutes:seconds - it doesn't report anything when any other secondary time unit is selected.
The current implementation made absolute sense back in the day when there was one time unit grouping measures.beats and minutes:seconds together, but since users can now select what they want their secondary unit to be, adding this would not hurt. Currently when you want to use 2 time units other than minutes:seconds, you have to go into the View menu all the time to toggle back and forth between the 2 of them.
To sum it up, support for the secondary time unit would be a time saver.

**Comment by jcsteh** [comment]
Oh neat. I wasn't aware of this. I see it was introduced in REAPER v5.982. I agree this should be done. Can't make any promises as to when though. :)

**Comment by MatejGolian** [comment]
Cool, thank you bloody much. :-)
